feature/home/src/main/java/com/epmedu/animeal/home/presentation/ui/HomeScreenUiMapbox.kt

```diff
@@ -0,0 +1,121 @@
+package com.epmedu.animeal.home.presentation.ui
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.statusBarsPadding
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.viewinterop.AndroidView
+import com.epmedu.animeal.foundation.switch.AnimealSwitch
+import com.epmedu.animeal.foundation.theme.bottomBarHeight
+import com.epmedu.animeal.home.presentation.model.FeedingPointUi
+import com.epmedu.animeal.home.presentation.model.MapLocation
+import com.epmedu.animeal.home.presentation.ui.map.MapBoxInitOptions
+import com.epmedu.animeal.home.presentation.ui.map.MapUiSettings
+import com.epmedu.animeal.home.presentation.ui.map.MarkerController
+import com.epmedu.animeal.home.presentation.ui.map.rememberMapInitOptions
+import com.epmedu.animeal.home.presentation.ui.map.rememberMapUiSettings
+import com.epmedu.animeal.home.presentation.ui.map.rememberMapViewWithLifecycle
+import com.epmedu.animeal.home.presentation.ui.map.setGesturesListener
+import com.epmedu.animeal.home.presentation.ui.map.setLocation
+import com.epmedu.animeal.home.presentation.viewmodel.HomeState
+import com.mapbox.geojson.Point
+import com.mapbox.maps.CameraOptions
+import com.mapbox.maps.MapView
+
+@Composable
+internal fun HomeScreenUiMapbox(
+    state: HomeState,
+    onFeedingPointSelect: (point: FeedingPointUi) -> Unit,
+    onGeolocationClick: (MapView) -> Unit,
+    onMapInteraction: () -> Unit,
+) = Box(modifier = Modifier.fillMaxSize()) {
+    val mapboxMapView = rememberMapboxMapView(homeState = state)
+
+    MapboxMap(
+        mapboxMapView = mapboxMapView,
+        state = state,
+        onFeedingPointClick = onFeedingPointSelect,
+        onMapInteraction = onMapInteraction,
+    )
+
+    AnimealSwitch(
+        modifier = Modifier
+            .statusBarsPadding()
+            .align(alignment = Alignment.TopCenter)
+            .padding(top = 24.dp),
+        onSelectTab = {}
+    )
+
+    GeoLocationFloatingActionButton(
+        modifier = Modifier
+            .padding(
+                bottom = bottomBarHeight + 24.dp,
+                end = 24.dp
```

Comment:
Looks like you can set common padding without specifying directions

Reply:
Well, yes I can, but this way is much easier to understand padding directions.